railties/railties.gemspec

```diff
@@ -44,6 +44,7 @@ Gem::Specification.new do |s|
   s.add_dependency "rake", ">= 12.2"
   s.add_dependency "thor", "~> 1.0"
   s.add_dependency "zeitwerk", "~> 2.6"
+  s.add_dependency "irb", "~> 1.6"
```

Comment:
I'm not sure how strict/loose this should be, so I selected the latest minor release. But I don't see a reason to let users specify anything older.

Reply:
More thoughts about this:

- If we set the upper bound, we'd restrict users' ability to upgrade `irb` in the given Rails version. Ideally, users should be able to upgrade to `2.0` if they feel like it.
- If we don't set the upper bound, users may unintentionally upgrade to a `irb` that has breaking change.

I think we could:

- Declare it as a dependency with: `>= 1.6`
- And add it to the Gemfile template with `gem "irb", "~> 1.6"`

WDYT?

Reply:
If we use any feature that only exists in irb 1.6 we should have this. Otherwise, just allow any version to be used.

Reply:
Make sense. Updated 👍